graspologic/embed/n2v.py

```diff
@@ -233,7 +238,13 @@ def __init__(
         inout_hyperparameter: float,
         random_state: Optional[np.random.RandomState] = None,
     ):
-        self.graph: nx.Graph = graph
+        self.original_graph: nx.Graph = graph
+
+        graph_with_new_ids, new_id_map = remap_node_ids(graph=graph)
```

Comment:
Create a representation of the graph where all original node ids are remapped to a new string node id, see utils.py